build_library/vm_image_util.sh

```diff
@@ -768,6 +768,15 @@ You can pass extra kernel parameters with -append, for example:
 When using -nographic or -serial you must also enable the serial console:
   ./$(basename "${script}") -nographic -append 'console=ttyS0,115200n8'
 EOF
+    local packetipxe="$(_dst_dir)/packet.ipxe"
```

Comment:
The ipxe script should have more clear name, like `flatcar_production_packet.ipxe`.

Reply:
done